contracts/build.rs

```diff
@@ -15,37 +15,27 @@ fn main() {
     // - https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath
     println!("cargo:rerun-if-changed=build.rs");
 
-    generate_contract("ERC20");
-    generate_contract("ERC20Mintable");
-    generate_contract_with_config("UniswapV2Router02", |builder| {
-        builder
-            .add_deployment_str(1, "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D")
-            .add_deployment_str(4, "0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D")
-            .add_deployment_str(100, "0x1C232F01118CB8B424793ae03F870aa7D0ac7f77")
-    });
-    generate_contract_with_config("UniswapV2Factory", |builder| {
-        builder
-            .add_deployment_str(1, "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f")
-            .add_deployment_str(4, "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f")
-            .add_deployment_str(100, "0xA818b4F111Ccac7AA31D0BCc0806d64F2E0737D7")
-    });
-    generate_contract("UniswapV2Pair");
-    // This is done to have a common interface for Sushiswap, Uniswap & Honeyswap
-    generate_contract("IUniswapLikeRouter");
-    generate_contract("IUniswapLikePair");
-    generate_contract_with_config("SushiswapV2Router02", |builder| {
+    generate_contract_with_config("BalancerV2Vault", |builder| {
         builder
-            .add_deployment_str(1, "0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F")
-            .add_deployment_str(4, "0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506")
-            .add_deployment_str(100, "0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506")
-    });
-    generate_contract_with_config("SushiswapV2Factory", |builder| {
-        builder
-            .add_deployment_str(1, "0xC0AEe478e3658e2610c5F7A4A2E1777cE9e4f2Ac")
-            .add_deployment_str(4, "0xc35DADB65012eC5796536bD9864eD8773aBc74C4")
-            .add_deployment_str(100, "0xc35DADB65012eC5796536bD9864eD8773aBc74C4")
+            .with_contract_mod_override(Some("balancer_v2_vault"))
```

Comment:
Can we just call this balance_v2 since we might wind up using more than just the vault. Also the vault is already in the contract name.

Reply:
The issue is that `ethcontract` doesn't support multiple contracts under the same module. Ideally, we would have:
```
mod balancer_v2 {
  type Vault;
  type WeighedPool;
}
```

But unfortunately, `ethcontract` gives us:
```
mod balancer_v2_vault {
  type Contract;
}
type BalancerV2Vault = balancer_v2_vault::Contract;

mod balancer_v2_weighted_pool {
  type Contract;
}
type BalancerV2WeightedPool = balancer_v2_weighted_pool::Contract;
```